src/js/events/MesosSummaryActions.js

```diff
@@ -64,7 +69,7 @@ function requestFromHistoryServer(resolve, reject, timeScale = 'last') {
       requestFromMesos(resolve, reject);
     },
     hangingRequestCallback() {
-      AppDispatcher.handleServerAction({type: REQUEST_MESOS_HISTORY_ONGOING});
+      AppDispatcher.handleServerAction({type: REQUEST_SUMMARY_HISTORY_ONGOING});
```

Comment:
This action type was never defined and therefore never handled (or called before for that matter)